frontend/README.md

```diff
@@ -25,6 +25,9 @@ To install the app, run the following commands in a command line:
 * `cd badgeyay/frontend`
 * `npm install`
 * `bower install`
+
+Tip: If you are encountering any errors while installation of dependencies, you can use `yarn install` instead of `npm install` after deleting and cloning the repository again.
```

Comment:
We don't want yarn, it's installation procedure should be documented as well if we use it. Let's stick to npm only for now

Reply:
Ok Sir will do so